assi2/css/style.css

```diff
@@ -0,0 +1,41 @@
+#parent{
+    display: flex;
+    flex-direction: column;
+    height: 300px;
+    /* background-color: aquamarine; */
+}
+
+.load-button-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.table-container{
+    display: flex;
+    justify-content: center;
+    /* align-items: center; */
+}
```

Comment:
add an enter - There should be proper formatting, no cluttered code would be accepted